src/lib/operators/table_scan.cpp

```diff
@@ -81,6 +83,8 @@ std::shared_ptr<const Table> TableScan::_on_execute() {
 
   const auto output_table = std::make_shared<Table>(in_table->column_definitions(), TableType::References);
 
+  _predicate = _resolve_uncorrelated_subqueries(_predicate);
```

Comment:
This means the visualization will not show the subselect anymore for the PQP, if done after query execution. Are we cool with this?

Reply:
Argh. Crap. Use `_resolved_predicate` internally?